engine/combat/utility/sos_consideration.py

```diff
@@ -50,9 +52,12 @@ def _default_appraisals(self: Self) -> list[Appraisal]:
     def _character_appraisals(self: Self) -> list[Appraisal]:
         match self.actor.character:
             case PlayerPartyCharacter.Zale:
-                return [Sunball()]
+                return [Sunball(value=100)]
```

Comment:
Value here is the utility weighting to make it always select this?

Reply:
Yes for now. The system knows it cant use things if it doesnt have the mp now.